.github/workflows/ci.yaml

```diff
@@ -0,0 +1,34 @@
+name: CI
+on: [push, pull_request, workflow_dispatch]
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout repo
```

Comment:
nit: "Checkout" or "Checkout Repository".

Is it required to have a name? IIUC you can just have uses, which names things cleanly.

Reply:
It's not required, but I think it makes the file nicer to read